components/Nav/index.tsx

```diff
@@ -76,7 +70,7 @@ const Nav: React.FC = () => {
   );
 };
 
-export async function getStaticProps() {
+export async function getServerProps() {
```

Comment:
`getServerProps` isn't a valid fetching function, however, this can be removed since it won't ever be called with it not being within a page component file.